dags/covid_etl_dag.py

```diff
@@ -110,6 +132,24 @@ def upload_to_snowflake(task_id):
             )
 
             return create_insert_task
+        
+        def qa_checks(**context):
+            try:
+                with open(qa_file, 'r') as fd:
+                    sqlfile = fd.read()
+                    sqllist = sqlfile.split(";")
+                    sf_hook = SnowflakeHook(snowflake_conn_id=Variable.get("SNOWFLAKE_CONNECTION", default_var="SNOWFLAKE"))
+                    for sql in sqllist:
+                        if len(strip(sql)) > 5:
+                            result = sf_hook.get_pandas_df(sql)
+                            if len(result.index) > 0:
```

Comment:
add check for GITUSER and skip the section in case it is not set (for the CI and local environments)